tests/test_job_management.py

```diff
@@ -1310,7 +1310,17 @@ def test_email_scws_list(dispatcher_live_fixture,
 
                 if 'resolve' in extracted_product_url:
                     print("need to resolve this:", extracted_product_url)
+
                     r = requests.get(extracted_product_url.replace('PRODUCTS_URL/dispatch-data', server))
+
+                    # this is to show how parameters can be overwritten in resolve; this never happens intentionally and is not dangerous
+                    # could be prevented
+                    alt_scw_list = ['066400220010.001', '066400230010.001']
+                    r_alt = requests.get(extracted_product_url.replace('PRODUCTS_URL/dispatch-data', server), params={'scw_list': alt_scw_list})
+                    alt_extracted_scw_list = parse_qs(parse.urlparse(r_alt.url).query)['scw_list'][0].split(",")
+                    assert alt_extracted_scw_list == alt_scw_list
+                    assert alt_extracted_scw_list != params['scw_list']
```

Comment:
Ok I see. 

Can we implement a check on the dispatcher-side to prevent this? Can we assume that when `resolve-job-url` is called, then only two (or three) params are expected? Those would be `job_id`, `session_id` and `token`?

Reply:
sure, but I am not even sure it is a bug. A bit of an unintended behavior. Easy to prevent.